src/handler.js

```diff
@@ -1,3 +1,3 @@
 const getATFLs = require('./functions/getATFsLambda')
 
-module.exports.getATFs = getATFLs
+module.exports.handler = getATFLs
```

Comment:
getATFLs -> we dont need technology used to be part of naming conventions for our constants. so Ls in getATFLs get be updated to something not specific to aws feature being used pls